lib/Doctrine/ruleset.xml

```diff
@@ -95,6 +95,8 @@
     <rule ref="SlevomatCodingStandard.ControlStructures.DisallowEqualOperators"/>
     <!-- Require language constructs without parentheses -->
     <rule ref="SlevomatCodingStandard.ControlStructures.LanguageConstructWithParentheses"/>
+    <!-- Require new instances with parentheses -->
+    <rule ref="SlevomatCodingStandard.ControlStructures.NewWithParenthesesSniff"/>
```

Comment:
@kukulich What is the correct Sniff name? Travis isn't find it :cry:

Reply:
@carusogabriel Remove "Sniff" at the end :)

```
<rule ref="SlevomatCodingStandard.ControlStructures.NewWithParentheses"/>
```